From c35467378232b1b21cca304f5dba37c687e51bca Mon Sep 17 00:00:00 2001
From: Andreas Zuber <zuber@puzzle.ch>
Date: Mon, 30 Oct 2017 09:52:32 +0100
Subject: [PATCH 2/3] import pyfa

Append $(python_get_sitedir)/pyfa to import path

Ensures that pyfa's import statements continue to work for systemwide
installation.
---
 pyfa.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/pyfa.py b/pyfa.py
index 4faa1edd..9cfa5e23 100755
--- a/pyfa.py
+++ b/pyfa.py
@@ -23,6 +23,7 @@ import os
 import platform
 import re
 import sys
+sys.path.append(os.path.join("%%SITEDIR%%", "pyfa"))
 import traceback
 from optparse import AmbiguousOptionError, BadOptionError, OptionParser
 
-- 
2.14.3

